lib/config/presets/github/index.ts

```diff
@@ -15,9 +16,15 @@ const http = new GithubHttp();
 export async function fetchJSONFile(
   repo: string,
   fileName: string,
-  endpoint: string
+  endpoint: string,
+  packageTag?: string
 ): Promise<Preset> {
-  const url = `${endpoint}repos/${repo}/contents/${fileName}`;
+  let ref = '';
+  if (is.nonEmptyString(packageTag)) {
+    ref = `?ref=${packageTag}`;
+  }
+  const url = `${endpoint}repos/${repo}/contents/${fileName}${ref}`;
+  logger.trace(`Preset URL is: ${url}`);
```

Comment:
Use logger metadata instead of string template
```suggestion
  logger.trace({ url }, `Preset URL`);
```